File: 27-remove-element/remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        #nums -> array, #val -> target val
        j = 0
        for i in range(len(nums)):
            if nums[i] != val:
                nums[j] = nums[i]
                j+=1

        return j 

File: 27-remove-element/remove-element.c
int removeElement(int* nums, int numsSize, int val) {
    //nums -> array, numsSize -> array size, val -> target value
    int j=0;
    for(int i=0; i<numsSize; i++)
    {
        if(nums[i] != val){
            nums[j++] = nums[i];
        }
    }
    return j;
}